src/views/about/about.jsx

```diff
@@ -72,12 +80,36 @@ var About = React.createClass({
                             <h3><FormattedMessage id='about.quotes' /></h3>
                             <img src="/images/about/quotes.gif" alt="Quotes about Scratch" />
                             <p><FormattedHTMLMessage id='about.quotesDescription' /></p>
+                            values={{
```

Comment:
remember to replace the FormattedHTMLMessage with just FormattedMessage.

Reply:
I think you'll need to go over all the FormattedHTMLMessages, changing them to `FormattedMessage` and double checking the values against the placeholder names.

Thanks for noticing and updating the other strings in the view! When you're all done you should be able to remove the `require(require('react-intl').FormattedHTMLMessage` on line 2 as well.

Reply:
👍